src/components/SettingDialog.vue

```diff
@@ -880,24 +891,34 @@ export default defineComponent({
     };
 
     const restartAllEngineProcess = () => {
-      store.dispatch("RESTART_ENGINES", { engineIds: store.state.engineIds });
+      store.dispatch("RESTART_ENGINES", { engineIds: engineIds.value });
     };
 
     const savingSetting = computed(() => store.state.savingSetting);
 
-    const samplingRateOptions: SavingSetting["outputSamplingRate"][] = [
-      "engineDefault",
-      24000,
-      44100,
-      48000,
-      88200,
-      96000,
+    const engineUseGpuOptions = [
+      { label: "CPU", value: false },
+      { label: "GPU", value: true },
     ];
-    const renderSamplingRateLabel = (
-      value: SavingSetting["outputSamplingRate"]
-    ) => {
+
+    const canEngineUseGpu = (engineId: string) => {
+      return store.getters.ENGINE_CAN_USE_GPU(engineId);
+    };
```

Comment:
ここ、「GPUが使えなければ変更不可」ではなく、「GPUが使えないならGPUに変更不可」のがユーザーに優しいかなと思いました！
なのでエンジン変更可能かどうかを制御すると良さそうに思いました。

こう？
```suggestion
    const canChangeUseGpu = computed(() => {
      // GPUが使えるエンジンの場合は変更可能
      // GPUが使えなくてもCPUには変更可能
      return (
        store.getters.ENGINE_CAN_USE_GPU(selectedEngineId.value) ||
        !engineUseGpu.value
      );
    });
```